webClient/src/app/editor/code-editor/code-editor.service.ts

```diff
@@ -19,6 +19,17 @@ export class CodeEditorService {
 
   closeFile(fileContext: ProjectContext) {
     this.editorControl.closeFileHandler(fileContext);
+    /* dequeue */
+    // this.log.warn(`code-editor.service ${fileContext.model.fileName}`);
```

Comment:
I can't remember why this is here.  It was model code for invocation of dequeue, which is now POST to datasetEnqueue.  Also, there are several closeFile functions here and in other repos.